test/use-cases.hs

```diff
@@ -0,0 +1,56 @@
+--
+-- Copyright © 2013-2015 Anchor Systems, Pty Ltd and Others
+--
+-- The code in this file, and the program it is a part of, is
+-- made available to you by its authors as open source software:
+-- you can redistribute it and/or modify it under the terms of
+-- the 3-clause BSD licence.
+--
+
+{-# LANGUAGE OverloadedStrings #-}
+
+module Main where
+
+import           Network.URI
+import           System.Environment     (getArgs, withArgs)
+import           System.FilePath.Posix  (splitFileName)
+import           Test.Hspec.WebDriver
+
+main :: IO ()
+main = do
+    args <- getArgs
+    case args of
+        (x:xs) | Just uri <- parseURI x -> withArgs xs . hspec $ spec uri
+        _ -> do
+            putStrLn "First argument must be a server URI."
+            putStrLn ""
+            putStrLn "This test relies on ./runit.sh, ./proxy.sh, and selenium being run"
+            putStrLn "To set up a selenium standalone server, download it here:"
+            putStrLn "http://docs.seleniumhq.org/download/"
+            putStrLn "Then run: java -jar selenium-server-standalone*.jar"
+
+spec :: URI -> Spec
+spec uri =
+    describe "As a user," $ do
+        session "I can create a token with a subset of my scope" $ using [Firefox, Chrome] $ do
```

Comment:
There is a way, defining custum instances for `TestCapabilities` and `Using`. An easy way would be to just use the `Capabilities` directly.